### PR TITLE
Fixed review text and updated Document validations

### PR DESCRIPTION
**Pull request by cameron-freshworks**
*Issue #:* /bcgov/entity#7145

*Description of changes:*

* Updated error msg for review page.

* Updated validations for Documents section.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

File: src/components/Summary/DocumentsDelivery.vue [hunk 1]
```diff
@@ -1,51 +1,53 @@
...
+            </v-col>
+            <v-col cols="9" class="px-0">
+              {{getBusinessContact.email || '(Not entered)'}}
+            </v-col>
+          </v-row>
+        </v-container>
+        <v-container v-if="isRoleStaff">
+          <v-row class="pl-4">
+            <v-col cols="3" class="px-0">
+              <label :class="{ 'error-text': documentDeliveryInvalid }"><strong>User Account</strong></label>
```
Comment (severinbeauvais) [inline src/components/Summary/DocumentsDelivery.vue:21]: Oh nice. That was easy :)

(PS Easy changes are a sign of well organized/written code. Kudos!)

**Review by severinbeauvais — APPROVED** [review]
LGTM.

Do you think it's worth updating the app version? If we merge this before my unit test PR then maybe not needed?

File: src/components/Summary/DocumentsDelivery.vue (hunk 1 above)
Comment (cameron-freshworks) [reply src/components/Summary/DocumentsDelivery.vue:21]: thanks! 
Exactly :)

**Comment by cameron-freshworks** [comment]
> LGTM.
> 
> Do you think it's worth updating the app version? If we merge this before my unit test PR then maybe not needed?

I considered it but did not, due to this sort of being a part of the last commit (although missed). 
Is this ok?

**Comment by severinbeauvais** [comment]
> > LGTM.
> > Do you think it's worth updating the app version? If we merge this before my unit test PR then maybe not needed?
> 
> I considered it but did not, due to this sort of being a part of the last commit (although missed).
> Is this ok?

The app version is for our use only - so we can tell what code the deployed app is running.

I think it's fine in this case. I will merge...

**Pull request merged**